main.go

```diff
@@ -60,6 +60,10 @@ func (l *Linter) String() string {
 func LinterFromName(name string) *Linter {
 	s := linterDefinitions[name]
 	parts := strings.SplitN(s, ":", 2)
+	if len(parts) < 2 {
+		kingpin.FatalUsage("invalid linter: %q", name)
```

Comment:
If that's preferable I can definitely update it to do that. I just thought showing the usage as well would be helpful as it's not uncommon for command line utilities to do that when invalid options are specified.